cladecombiner/taxon_utils.py

```diff
@@ -36,6 +36,11 @@ def read_taxa(
     Sequence[Taxon]
         Container of the taxa as Taxon objects.
     """
+    if nomenclature is not None:
```

Comment:
maybe `assert nomenclature is None or isintance(...)`

Reply:
Fair

Reply:
Fixed in #13, as well as the fact that these were accidentally both checks of `nomenclature`